alpha/lib/model/DeliveryProfileLive.php

```diff
@@ -410,6 +410,9 @@ protected function getRenderer($flavors)
 	protected function filterStreamIdsByBitrate ($streams)
 	{
 		$streamIds = array();
+		
+		$this->sanitizeRequestedFlavorParamsIds($streams);
```

Comment:
Move this call after validating streams exist , instead of validating it twice inside. Also ince move , the validation can be removed from internal protected function

Reply:
resolved.